qiskit/algorithms/optimizers/gsls.py

```diff
@@ -121,6 +121,8 @@ def minimize(
             var_lb = np.array([l for (l, _) in bounds])
             var_ub = np.array([u for (_, u) in bounds])
 
+        # TODO: ls_optimize last element of return tuple is norm of gradient,
+        #  not number of gradient evaluations
         x, fun, nfev, njev = self.ls_optimize(x0.size, fun, x0, var_lb, var_ub)
```

Comment:
It looks like the last variable shouldn't be `njev` (and should not be put in `njev` field of result)

Reply:
Yep, you're right 👍🏻